spec/main_spec.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../lib/board.rb'
+require_relative '../lib/engine.rb'
+
+#  --- VScode setup ---
+# require 'rspec/autorun'
+# RSpec.describe Enumerable do
+#  --- VScode setup end ---
+
+describe Engine do
+  let(:game1) { Engine.new }
+  let(:field_win) { ['x', 'x', 'x', 4, 5, 6, 7, 8, 9] }
+
+  describe '#Class Board' do
+    context 'Display Board' do
+      it 'Should display the board' do
```

Comment:
`[Optional]` It's best practice to start the `it` block info with a singular verb. For example:
```ruby
it 'displays the board' do
end
it 'updates the board when player 1 makes a move' do
end
it 'validates the first valid input' do
end
```